k8s/jaeger-operator/Makefile

```diff
@@ -8,10 +8,10 @@ VERIFY_WAIT_TIMEOUT = 1200
 CHART_NAME := jaeger-operator
 APP_ID ?= $(CHART_NAME)
 
-SOURCE_REGISTRY ?= marketplace.gcr.io/google
```

Comment:
Revert this file if it is not related to the kalm deprecation.

Reply:
Done